ERROR - 2024-05-05 09:34:19 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'crud', 3306, '', 0)
#1 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\capstonee\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\capstonee\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\capstonee\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\capstonee\backend\app\Controllers\MainController.php(695): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(942): App\Controllers\MainController->login()
#8 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
#9 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\capstonee\backend\public\index.php(86): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\capstonee\backend\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\capstonee\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\capstonee\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\capstonee\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\capstonee\backend\app\Controllers\MainController.php(695): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(942): App\Controllers\MainController->login()
#7 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
#8 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\capstonee\backend\public\index.php(86): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\capstonee\backend\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#11 {main}
CRITICAL - 2024-05-05 09:34:19 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(662): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\MainController.php(695): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\MainController->login()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\capstonee\\backend\\public\\index.php')
ERROR - 2024-05-05 09:34:23 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'crud', 3306, '', 0)
#1 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\capstonee\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\capstonee\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\capstonee\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\capstonee\backend\app\Controllers\MainController.php(695): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(942): App\Controllers\MainController->login()
#8 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
#9 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\capstonee\backend\public\index.php(86): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\capstonee\backend\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\capstonee\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\capstonee\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\capstonee\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\capstonee\backend\app\Controllers\MainController.php(695): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(942): App\Controllers\MainController->login()
#7 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
#8 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\capstonee\backend\public\index.php(86): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\capstonee\backend\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#11 {main}
CRITICAL - 2024-05-05 09:34:23 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(662): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\MainController.php(695): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\MainController->login()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\capstonee\\backend\\public\\index.php')
ERROR - 2024-05-05 09:34:36 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'crud', 3306, '', 0)
#1 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\capstonee\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\capstonee\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\capstonee\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\capstonee\backend\app\Controllers\MainController.php(695): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(942): App\Controllers\MainController->login()
#8 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
#9 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\capstonee\backend\public\index.php(86): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\capstonee\backend\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\capstonee\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\capstonee\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\capstonee\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\capstonee\backend\app\Controllers\MainController.php(695): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(942): App\Controllers\MainController->login()
#7 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
#8 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\capstonee\backend\public\index.php(86): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\capstonee\backend\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#11 {main}
CRITICAL - 2024-05-05 09:34:36 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(662): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\MainController.php(695): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\MainController->login()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\capstonee\\backend\\public\\index.php')
ERROR - 2024-05-05 09:34:42 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'crud', 3306, '', 0)
#1 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\capstonee\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\capstonee\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\capstonee\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\capstonee\backend\app\Controllers\MainController.php(695): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(942): App\Controllers\MainController->login()
#8 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
#9 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\capstonee\backend\public\index.php(86): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\capstonee\backend\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\capstonee\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\capstonee\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\capstonee\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\capstonee\backend\app\Controllers\MainController.php(695): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(942): App\Controllers\MainController->login()
#7 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
#8 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\capstonee\backend\public\index.php(86): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\capstonee\backend\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#11 {main}
CRITICAL - 2024-05-05 09:34:42 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(662): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\MainController.php(695): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\MainController->login()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\capstonee\\backend\\public\\index.php')
ERROR - 2024-05-05 09:34:57 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'crud', 3306, '', 0)
#1 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\capstonee\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\capstonee\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\capstonee\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\capstonee\backend\app\Controllers\MainController.php(695): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(942): App\Controllers\MainController->login()
#8 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
#9 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\capstonee\backend\public\index.php(86): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\capstonee\backend\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\capstonee\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\capstonee\backend\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\capstonee\backend\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\capstonee\backend\app\Controllers\MainController.php(695): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(942): App\Controllers\MainController->login()
#7 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
#8 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\capstonee\backend\public\index.php(86): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\capstonee\backend\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#11 {main}
CRITICAL - 2024-05-05 09:34:58 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(662): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\MainController.php(695): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\MainController->login()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\capstonee\\backend\\public\\index.php')
ERROR - 2024-05-05 09:45:33 --> mysqli_sql_exception: Unknown column 'cart_status' in 'where clause' in C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\laragon\www\capstonee\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\capstonee\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\capstonee\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\capstonee\backend\app\Controllers\MainController.php(239): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(942): App\Controllers\MainController->getCart('102')
#8 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
#9 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\capstonee\backend\public\index.php(86): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\capstonee\backend\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#12 {main}
CRITICAL - 2024-05-05 09:45:33 --> Unknown column 'cart_status' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `cart`
WHERE `id` = :id:
AND `cart_status` != :cart_status:', [...], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\MainController.php(239): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\MainController->getCart('102')
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\capstonee\\backend\\public\\index.php')
ERROR - 2024-05-05 09:50:26 --> mysqli_sql_exception: Unknown column 'cart_status' in 'field list' in C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\laragon\www\capstonee\backend\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\laragon\www\capstonee\backend\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\capstonee\backend\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\capstonee\backend\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\capstonee\backend\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\capstonee\backend\app\Controllers\MainController.php(798): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(942): App\Controllers\MainController->Cart()
#10 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
#11 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\capstonee\backend\public\index.php(86): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\capstonee\backend\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#14 {main}
CRITICAL - 2024-05-05 09:50:26 --> Unknown column 'cart_status' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cart` (`id`, `shop_id`, `quantity`, `cart_status`) VALUES (:id:, :shop_id:, :quantity:, :cart_status:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\MainController.php(798): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\MainController->Cart()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\capstonee\\backend\\public\\index.php')
ERROR - 2024-05-05 09:50:40 --> mysqli_sql_exception: Unknown column 'cart_status' in 'field list' in C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\laragon\www\capstonee\backend\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\laragon\www\capstonee\backend\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\capstonee\backend\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\capstonee\backend\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\capstonee\backend\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\capstonee\backend\app\Controllers\MainController.php(798): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(942): App\Controllers\MainController->Cart()
#10 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
#11 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\capstonee\backend\public\index.php(86): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\capstonee\backend\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#14 {main}
CRITICAL - 2024-05-05 09:50:40 --> Unknown column 'cart_status' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cart` (`id`, `shop_id`, `quantity`, `cart_status`) VALUES (:id:, :shop_id:, :quantity:, :cart_status:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\MainController.php(798): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\MainController->Cart()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\capstonee\\backend\\public\\index.php')
ERROR - 2024-05-05 09:50:49 --> mysqli_sql_exception: Unknown column 'cart_status' in 'field list' in C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\laragon\www\capstonee\backend\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\laragon\www\capstonee\backend\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\capstonee\backend\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\capstonee\backend\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\capstonee\backend\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\capstonee\backend\app\Controllers\MainController.php(798): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(942): App\Controllers\MainController->Cart()
#10 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
#11 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\capstonee\backend\public\index.php(86): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\capstonee\backend\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#14 {main}
CRITICAL - 2024-05-05 09:50:49 --> Unknown column 'cart_status' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cart` (`id`, `shop_id`, `quantity`, `cart_status`) VALUES (:id:, :shop_id:, :quantity:, :cart_status:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\MainController.php(798): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\MainController->Cart()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\capstonee\\backend\\public\\index.php')
ERROR - 2024-05-05 09:51:00 --> mysqli_sql_exception: Unknown column 'cart_status' in 'field list' in C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\laragon\www\capstonee\backend\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\laragon\www\capstonee\backend\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\capstonee\backend\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\capstonee\backend\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\capstonee\backend\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\capstonee\backend\app\Controllers\MainController.php(798): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(942): App\Controllers\MainController->Cart()
#10 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
#11 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\capstonee\backend\public\index.php(86): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\capstonee\backend\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#14 {main}
CRITICAL - 2024-05-05 09:51:00 --> Unknown column 'cart_status' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cart` (`id`, `shop_id`, `quantity`, `cart_status`) VALUES (:id:, :shop_id:, :quantity:, :cart_status:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\MainController.php(798): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\MainController->Cart()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\capstonee\\backend\\public\\index.php')
ERROR - 2024-05-05 09:51:20 --> mysqli_sql_exception: Unknown column 'cart_status' in 'field list' in C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\capstonee\backend\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\laragon\www\capstonee\backend\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\laragon\www\capstonee\backend\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\laragon\www\capstonee\backend\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\capstonee\backend\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\capstonee\backend\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\capstonee\backend\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\capstonee\backend\app\Controllers\MainController.php(798): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(942): App\Controllers\MainController->Cart()
#10 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
#11 C:\laragon\www\capstonee\backend\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\capstonee\backend\public\index.php(86): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\capstonee\backend\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#14 {main}
CRITICAL - 2024-05-05 09:51:20 --> Unknown column 'cart_status' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cart` (`id`, `shop_id`, `quantity`, `cart_status`) VALUES (:id:, :shop_id:, :quantity:, :cart_status:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\MainController.php(798): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\MainController->Cart()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\capstonee\\backend\\public\\index.php')
CRITICAL - 2024-05-05 15:05:30 --> Undefined property: stdClass::$cottage_id
in APPPATH\Controllers\MainController.php on line 48.
 1 APPPATH\Controllers\MainController.php(48): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$cottage_id', 'C:\\laragon\\www\\capstonee\\backend\\app\\Controllers\\MainController.php', 48)
 2 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\MainController->cottageBooking()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\capstonee\\backend\\public\\index.php')
CRITICAL - 2024-05-05 15:22:40 --> Undefined property: stdClass::$cottage_id
in APPPATH\Controllers\MainController.php on line 48.
 1 APPPATH\Controllers\MainController.php(48): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$cottage_id', 'C:\\laragon\\www\\capstonee\\backend\\app\\Controllers\\MainController.php', 48)
 2 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\MainController->cottageBooking()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\capstonee\\backend\\public\\index.php')
CRITICAL - 2024-05-05 15:50:57 --> Undefined property: stdClass::$cottage_id
in APPPATH\Controllers\MainController.php on line 48.
 1 APPPATH\Controllers\MainController.php(48): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$cottage_id', 'C:\\laragon\\www\\capstonee\\backend\\app\\Controllers\\MainController.php', 48)
 2 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\MainController->cottageBooking()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MainController))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\capstonee\\backend\\public\\index.php')
